### PR TITLE
Simplify base resource schema

### PR DESCRIPTION
**Pull request by leplatrem**
Fixes https://github.com/mozilla-services/readinglist/issues/170

### CODE REVIEW

**Comment by leplatrem** [comment]
@ametaireau r?

File: cliquet/resource.py [hunk 1]
```diff
@@ -330,9 +330,8 @@ def update_record(self, record, old=None):
             if nothing_changed:
                 return new
 
-        record_id = new[self.id_field]
         try:
-            return self.db.update(record_id=record_id,
+            return self.db.update(record_id=self.record_id,
```
Comment (ametaireau) [inline cliquet/resource.py]: Are we sure `self.record_id == new[self.id_field]` ? We're kinda bypassing the information provided in the `new` object if we're looking at what's in `self.record_id`.

File: cliquet/tests/resource/test_views.py [hunk 2]
```diff
@@ -126,6 +126,46 @@ def test_replace_with_invalid_record_returns_400(self):
...
+
+    def test_last_modified_is_not_validated_and_overwritten(self):
+        record = MINIMALIST_RECORD.copy()
+        record['last_modified'] = 'abc'
+        resp = self.app.post_json(self.collection_url,
+                                  record,
+                                  headers=self.headers)
+        self.assertNotEqual(resp.json['last_modified'], 'abc')
+
+    def test_modify_works_with_invalid_last_modified(self):
```
Comment (ametaireau) [inline cliquet/tests/resource/test_views.py]: ignores_last_modified ?

**Comment by ametaireau** [comment]
Looks good to me.

**Pull request merged**